src/libNetwork/RumorManager.h

```diff
@@ -0,0 +1,68 @@
+#ifndef ZILLIQA_RUMORMANAGER_H
```

Comment:
Following our header guard naming convention, this should be __RUMOR_MANAGER_H__.
Please update the other files similarly.

Reply:
Fixed